test/unit/modules/observer/observer.spec.js

```diff
@@ -316,6 +316,21 @@ describe('Observer', () => {
     }).then(done)
   })
 
+  it('Cannot enable reactivity on a property that is already defined', done => {
+    const vm = new Vue({
+      data: {
+        person: {
+          age: 32
+        }
+      }
+    }).$mount()
+    vm.person.job = 'Programmer'
+    Vue.set(vm.person, 'job', 'Educator')
+    waitForUpdate(() => {
+      expect(`Cannot enable reactivity on a property that is already defined`).toHaveBeenWarned()
+    }).end(done)
```

Comment:
I think this should be `then` rather than `end`.

Reply:
I've tried that as well, but the result is the same.